### PR TITLE
#3161 Add a test to verify the node dialog

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3161)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/e2e/nodeDialog.test.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+    // press "ESC" back to node list
+    cy.get('body:visible').trigger('keydown', { keyCode: 27, which: 27 });
+    // press "ESC" again back to home page
+    cy.findAllByText(/^nodes$/i)
+      .first()
+      .trigger('keydown', { keyCode: 27, which: 27 });
+
+    cy.findByTitle('Create a new workspace').click();
+    cy.findByText(/^quick\sstart$/i)
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/nodeDialog.test.js]: Hmm,  but now selector like this is a bit hard to read. I wonder if we can get rid of the escape char... 🤔

File: ohara-manager/client/cypress/e2e/nodeDialog.test.js [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+    // Create a workspace for this node
+    // Step1: workspace name (using default)
+    cy.findAllByText(/^next$/i)
+      .filter(':visible')
+      .click();
+
+    // Step2: select nodes
+    cy.contains('button', 'Select nodes').click();
+    cy.findByText(new RegExp(`^${nodeHost}$`, 'i'))
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/nodeDialog.test.js]: Also this

File: ohara-manager/client/cypress/e2e/nodeDialog.test.js (hunk 1 above)
Comment (saivirtue) [reply ohara-manager/client/cypress/e2e/nodeDialog.test.js]: I think that's why I use ignore-case in some place only...It's hard to read in some situations.

File: ohara-manager/client/cypress/e2e/nodeDialog.test.js (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/client/cypress/e2e/nodeDialog.test.js]: Maybe variable cases could use case-sensitive only?

**Comment by saivirtue** [comment]
retry

**Comment by saivirtue** [comment]
retry manager

**Comment by saivirtue** [comment]
retry manager-e2e

**Pull request merged**